packages/typescript-estree/src/create-program/createProjectProgram.ts

```diff
@@ -40,19 +41,11 @@ function createProjectProgram(
     return astAndProgram;
   }
 
-  const describeFilePath = (filePath: string): string => {
-    const relative = path.relative(
-      parseSettings.tsconfigRootDir || process.cwd(),
-      filePath,
-    );
-    if (parseSettings.tsconfigRootDir) {
-      return `<tsconfigRootDir>/${relative}`;
-    }
-    return `<cwd>/${relative}`;
-  };
+  const describeProjectFilePath = (projectFile: string): string =>
+    describeFilePath(projectFile, parseSettings.tsconfigRootDir);
 
-  const describedFilePath = describeFilePath(parseSettings.filePath);
-  const relativeProjects = parseSettings.projects.map(describeFilePath);
+  const describedFilePath = describeProjectFilePath(parseSettings.filePath);
```

Comment:
wdyt about removing `describeProjectFilePath` anonyms function form call stack?
```suggestion
  const describedFilePath = describeFilePath(parseSettings.filePath, parseSettings.tsconfigRootDir);
```

Reply:
Heh, yeah, works for me.